src/platform_impl/ios/view.rs

```diff
@@ -26,6 +27,12 @@ pub struct WinitViewState {
     pinch_gesture_recognizer: RefCell<Option<Id<UIPinchGestureRecognizer>>>,
     doubletap_gesture_recognizer: RefCell<Option<Id<UITapGestureRecognizer>>>,
     rotation_gesture_recognizer: RefCell<Option<Id<UIRotationGestureRecognizer>>>,
+    pan_gesture_recognizer: RefCell<Option<Id<UIPanGestureRecognizer>>>,
+
+    // for iOS delta references the start of the Gesture
+    rotation_last_delta: RefCell<Option<f64>>,
+    pinch_last_delta: RefCell<Option<f64>>,
+    pan_last_delta: RefCell<Option<CGPoint>>,
```

Comment:
Nit: Use `CGFloat` instead of `f64` (to support 32-bit iOS, which is currently the only way for me to test this on a real device).

Also, use `Cell` instead of `RefCell` here, that's both easier to work with, and more performant (but only possible because `Option<CGFloat>` is `Copy`).

And finally, maybe avoid the `Option` and just use `0.0`/`CGPoint::new(0.0, 0.0)` as the default value?

Reply:
@madsmtm sure I can change to CGFloat. I wonder though, there are many `f64`s and `u64`s in `src/event.rs`, so somehow those don't break your 32-bit iOS device?

I don't mind changing the `Option`s to zeros, but as a Rust noob my understanding was that `Option` is preferable for representing null values?

Reply:
32-bit devices can _use_ `f64`/`u64`, it's just less performant. The issue is more that `CGFloat` is defined as `f32` on 32-bit devices, and `f64` on 64-bit devices, and it's what Apple's APIs return, so you need to either use `CGFloat` too, or do a cast to ensure that the compiler doesn't complain.

I think you can test that it compiles yourself with `cargo +nightly check -Zbuild-std --target=armv7s-apple-ios`.